scripts/resender.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python3
+
+import click
+import requests
+from ape import project
+from ape.api import AccountAPI
+from ape.cli import ConnectedProviderCommand, account_option
+from ape.contracts import ContractInstance
+from ape.logging import logger
+
+
+def get_release_events(graphql_endpoint: str) -> list[dict]:
+    """
+    Queries GraphQL endpoint to retrieve all new `Release` events
+    on Polygon network
+    """
+
+    gql = (
+        """
+    query ReleasetoResend {
+    releaseds(
+        where: {releaseTxSender: "0x0000000000000000000000000000000000000000", releaseResent: false}
+        ) {
+            id
+        }
+    }
+    """
+    )
+
+    s = requests.session()
+    s.headers = {"Accept": "application/json", "Content-Type": "application/json"}
+
+    response = s.post(graphql_endpoint, json={"query": gql})
+
```

Comment:
I would add here some basic checks in case the subgraph is down or the URL is not correct 
```suggestion

    if response.status_code != 200:
        raise Exception("GraphQL endpoint not reachable")

```

Reply:
👍  - maybe also include "response.text" in the exception?

Reply:
✔️